packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/resources/PersonResource.java

```diff
@@ -0,0 +1,101 @@
+package software.uncharted.terarium.hmiserver.resources;
+
+import io.quarkus.security.Authenticated;
+import org.eclipse.microprofile.openapi.annotations.tags.Tag;
+import org.eclipse.microprofile.rest.client.inject.RestClient;
+import software.uncharted.terarium.hmiserver.models.dataservice.Association;
+import software.uncharted.terarium.hmiserver.models.dataservice.Person;
+import software.uncharted.terarium.hmiserver.proxies.PersonProxy;
+
+import javax.inject.Inject;
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+@Path("/api/persons")
+@Authenticated
+@Produces(MediaType.APPLICATION_JSON)
+@Tag(name = "Person REST Endpoints")
+public class PersonResource {
+
+	@Inject
+	@RestClient
+	PersonProxy proxy;
+
+	@GET
+	@Path("/associations/{id}")
+	public Response getAssociation(
+		@PathParam("id") final String id
+	) {
+		return proxy.getAssociation(id);
+	}
+
+	@DELETE
+	@Path("/associations/{id}")
+	public Response deleteAssociation(
+		@PathParam("id") final String id
+	) {
+		return proxy.deleteAssociation(id);
+	}
+
+	@PATCH
+	@Path("/associations/{id}")
+	@Consumes(MediaType.APPLICATION_JSON)
+	public Response updateAssociation(
+		@PathParam("id") final String id,
+		final Association association
+	) {
+		return proxy.updateAssociation(id, association);
+	}
+
+	@POST
+	@Path("/associations")
+	@Consumes(MediaType.APPLICATION_JSON)
+	public Response createAssociation(
+		final Association association
+	) {
+		return proxy.createAssociation(association);
+	}
+
+	@GET
+	public Response getPersons(
+		@QueryParam("page_size") final Integer pageSize,
+		@QueryParam("page") final Integer page
+	) {
+		return proxy.getPersons(pageSize, page);
+	}
+
+	@POST
+	@Consumes(MediaType.APPLICATION_JSON)
+	public Response createPerson(
+		final Person person
+	) {
+		return proxy.createPerson(person);
+	}
+
+	@GET
+	@Path("/{id}")
+	public Response getPerson(
+		@PathParam("id") final String id
+	) {
+		return proxy.getPerson(id);
+	}
```

Comment:
I don't see an integration with _Keyclock_ PostGres Table.
Is that something we will do in another PR?

Reply:
this is data-service wiring only

Reply:
I thought the Proxy was the wiring
```
hmi-client -> hmi-server (/api/* endpoints) -> hmi-server (proxy to DS) -> data-service
```

Reply:
yeah, I'm only wiring up endpoints that come for the data service. Currently, the data service has "Person" endpoints. I'm sure this will change later on, and as far as I know, there is no mock data for this endpoint anyway.

Reply:
@YohannParis the Person endpoint is for managing info about people related to a resource that aren't necessarily users (e.g. I uploaded this Dataset but the maintainer is actually some scientist in the UK). 

The script is populating the endpoint with one entry: 
```
        {
            "name": "Adam Smith",
            "email": "Adam@test.io",
            "org": "Uncharted",
            "website": "",
            "is_registered": True,
        }
```